src/main/java/net/mcreator/element/converter/ConverterRegistry.java

```diff
@@ -43,6 +44,7 @@
 		put(ModElementType.BIOME, Collections.singletonList(new BiomeSpawnListConverter()));
 		put(ModElementType.OVERLAY, Collections.singletonList(new OverlayCoordinateConverter()));
 		put(ModElementType.BIOME, Collections.singletonList(new BiomeDefaultFeaturesConverter()));
+		put(ModElementType.PROCEDURE, Collections.singletonList(new ProcedureStringsFixer()));

```

Comment:
You can't put the same element twice, you are overriding old procedure entry. Learn how Maps work in Java here: https://docs.oracle.com/javase/8/docs/api/java/util/Map.html

Just add to the existing list of entries under the procedure key.

Reply:
The commit is called "Started ..." for a reason, it's not done yet I'm waiting for Goldorion's help. I would like to know, if we can't put the same element twice, why did you do it for the Biomes thing then.

Reply:
@Goldorion added the other one and I did not notice this. I will fix this. You can't do this way. Check how Maps and their keys work.

Reply:
No it's okay I'll fix it, I'm editing the file so it would avoid problems between us.

Reply:
The fix for the biome does not belong to this PR. Keep things organized.

I fixed it in this commit: https://github.com/MCreator/MCreator/commit/62d8df4f469da9033e59352bab661b98e04062bb

Reply:
Fixed.